test/clj/chronograph/handlers/organization_test.clj

```diff
@@ -91,3 +99,48 @@
         (is (s/valid? :organizations/organization
                       (:body response))
             "Slug length can be exactly 256 characters at most.")))))
+
+
+(deftest find-one-organization
+  (testing "when fetching an organization"
+    (let [user (factories/create-user)
+          slug "bar"
+          organization (:body (create-organization
+                               {:body {:name "foo" :slug slug}
+                                :user user}))]
+      (is (= {:status 200
+              :headers {}
+              :body organization}
+             (find-one {:params {:slug slug}
+                        :user user}))
+          "Fetching an organization with authorized user returns the organization details.")
+      (is (= {:status 401
+              :headers {}
+              :body {:error "Unauthorized"}}
+             (find-one {:params {:slug slug}
+                        :user nil}))
+          "Fetching an organization with unauthorized user fails with HTTP error.")
+      (is (= {:status 400
+              :headers {}
+              :body {:error "Bad slug"}}
+             (find-one {:params {:slug nil}
+                        :user user}))
+          "Fetching an organization with invalid slug fails with HTTP error.")
+      (is (= {:status 404
+              :headers {}
+              :body {:error "Not found"}}
+             (find-one {:params {:slug "i-dont-exist"}
```

Comment:
replace "i-dont-exist" with a uuid-string so slug's nonexistence is guaranteed

Reply:
done

Reply:
You could actually leave it as a readable sentinel value like "non-existent-slug" etc, since in the context of the test it makes sense. But either ways is fine.